docs/application/dotnet/guides/opentk/work-with-skiasharp.md

```diff
@@ -0,0 +1,183 @@
+```C#
+private IntPtr pBitMap;                   // memory handle
+private int rowByte;                      // bytes of one row 
+private int bitmapHeight, bitmapWidth;    // height and width of the canvas
+
+private void CreateBitmap()
+{
+    bitmapHeight = (int)(skiaPercent * mainWindow.Height);
+    bitmapWidth = (int)(skiaPercent * mainWindow.Width);
+    pBitMap = Marshal.AllocHGlobal(bitmapWidth * bitmapHeight * 4);     // allocate a memory block
+
+    rowByte = bitmapWidth * 4;
+}
+
+private void FreeBitmap()
+{
+    if (pBitMap != IntPtr.Zero)
+    {
+        Marshal.FreeHGlobal(pBitMap);   // free the allocated memory block
+        pBitMap = IntPtr.Zero;
+    }
+}
+```
+
+## Create SKCanvas on memory block
+
+Create SKSurface on the memory block, and then you can get the SKCanvas from the SKSurface.
+```C#
+private SKSurface surface;
+private SKCanvas canvas;
+
+private void CreateSKCanvas()
+{
+    GL.PixelStore(All.UnpackAlignment, 4);
+    
+    // create the surface
+    var info = new SKImageInfo(bitmapWidth, bitmapHeight, SKImageInfo.PlatformColorType, SKAlphaType.Premul);
+    surface = SKSurface.Create(info, pBitMap, rowByte);
+    if (surface != null)
+    {
+        canvas = surface.Canvas;
+    }
+}
+```
+
+## Draw text on the SKCanvas
+
+```C#
+private void DrawTextBySkiaSharp(SKCanvas canvas, int canvasWidth, int canvasHeight)
+{
+    canvas.DrawColor(SKColors.White);                                   // clear the canvas as white
+
+    using (var paint = new SKPaint())
+    {
+        paint.TextSize = 64.0f;
+        paint.IsAntialias = true;
+        paint.Color = SKColors.Red;
+        paint.IsStroke = false;
+        paint.TextAlign = SKTextAlign.Center;
+
+        canvas.DrawText("SkiaSharp", width / 2f, 144.0f, paint);        // Draw text 'SkiaSharp' on the canvas
+    }
+}
+
+private void DrawTextOnSkCanvas()
+{
+    if (canvas != null)
+    {
+        DrawTextBySkiaSharp(canvas, bitmapWidth, bitmapHeight);
+        canvas.Flush();
+    }
+}
+```
+
+## Create 2D texture from memory block.
+
+Call `GL.TexImage2D` of `OpenTK.Graphics.ES20` to generate 2D texture from the memory block.
+```C#
+private void Create2DTextureFromMemory()
+{
+    GL.TexImage2D(
+                All.Texture2D,
+                0,
+                All.BgraExt,
+                bitmapWidth,
+                bitmapHeight,
+                0,
+                All.BgraExt,
+                All.UnsignedByte,
+                pBitMap
+            );
+
+    GL.TexParameter(All.Texture2D, All.TextureMinFilter, (float)All.LinearMipmapLinear);
+    GL.TexParameter(All.Texture2D, All.TextureMagFilter, (float)All.Linear);
+    GL.GenerateMipmap(All.Texture2D);
+}
+```
+
+## Draw 2D Texture
+Draw a 2D texture with graphics APIs of OpenTK, for more information you can reference to [OpenTK samples](https://github.sec.samsung.net/dotnet/opentk_samples).
```

Comment:
Please change to : 
Draw a 2D texture with graphics APIs of OpenTK, for more information, see [OpenTK samples](https://github.sec.samsung.net/dotnet/opentk_samples).